README.md

```diff
@@ -20,8 +20,7 @@ Install the plugin in WordPress. You can download a
 plugin uploader ("Plugins" > "Add New" > "Upload Plugin").
 
 ## Non-English Usage
-Safe Redirect Manager is available in English, French, and Slovak. Instructions for translating the plugin into other
-languages are below.
+Safe Redirect Manager is available in English and other languages.  A listing of those languages and instructions for translating the plugin into other languages is available on [Translating WordPress](https://translate.wordpress.org/projects/wp-plugins/safe-redirect-manager/).  Many thanks to the [contributors on the translation teams](https://translate.wordpress.org/projects/wp-plugins/safe-redirect-manager/contributors/)!
```

Comment:
```suggestion
A listing of languages Safe Redirect Manager is available in beyond English and instructions for translating the plugin into more languages is available on [Translating WordPress](https://translate.wordpress.org/projects/wp-plugins/safe-redirect-manager/).  Many thanks to the [contributors on the translation teams](https://translate.wordpress.org/projects/wp-plugins/safe-redirect-manager/contributors/)!
```

Reply:
I feel like my edit is still structurally awkward, not sure how to fix it at the moment but the current initial sentence is also strange with the heading IMO.